reconcile/gabi_authorized_users.py

```diff
@@ -54,11 +66,11 @@ def fetch_desired_state(
     gabi_instances: Iterable[Mapping], ri: ResourceInventory
 ) -> None:
     for g in gabi_instances:
-        exp_date = datetime.strptime(g["expirationDate"], "%Y-%m-%d").date()
-        if (exp_date - date.today()).days > EXPIRATION_MAX:
+        expiration_date = datetime.strptime(g["expirationDate"], "%Y-%m-%d").date()
+        if (expiration_date - date.today()).days > EXPIRATION_DAYS_MAX:
             raise RunnerException(
-                f'The maximum expiration date of {g["name"]} '
-                f"shall not exceed {EXPIRATION_MAX} days form today"
+                f'The maximum expiration date of {g["name"]} shall not '
+                f"exceed {EXPIRATION_DAYS_MAX} days from {expiration_date}"
```

Comment:
```suggestion
                f"exceed {EXPIRATION_DAYS_MAX} days from today"
```

Reply:
As part of some other changes, this was a different message, which included more details to make it more informative. I then dropped some patches to make fewer changes in this series and must have botched rebase following fixing a conflict. Sorry about this!

Reply:
Done. Hunk dropped from d322978e3eb8bef89e619799812d604fb2537bcc.

Reply:
thanks for fixing